lib/resource.nunjucks

```diff
@@ -132,26 +132,40 @@
                       <h3>Body</h3>
                       {% for key, b in method.body %}
                         <p><strong>Media Type: {{ key }}</strong></p>
-                        {% if b.formParameters %}
+                        {% if b.formParameters or b.type and not b.schema %}
```

Comment:
I think we all are not sure what's the best way, currently inline is displayed using the items template which is fine. Next should be references to global or library definitions.